xbmc/pvr/epg/EpgInfoTag.cpp

```diff
@@ -226,10 +226,11 @@ CDateTime CPVREpgInfoTag::GetCurrentPlayingTime() const
       CServiceBroker::GetPVRManager().Clients()->IsTimeshifting())
   {
     // timeshifting; start time valid?
-    time_t startTime = CServiceBroker::GetDataCacheCore().GetStartTime();
-    if (startTime > 0)
+    int64_t playTime = CServiceBroker::GetDataCacheCore().GetPlayTime();
+    if (playTime > 0)
     {
-      return CDateTime(startTime + CServiceBroker::GetDataCacheCore().GetPlayTime() / 1000);
+      time_t startTime = CServiceBroker::GetDataCacheCore().GetStartTime();
+      return CDateTime(startTime + playTime / 1000);
```

Comment:
That looks suspicious... shouldn't this read `return CDateTime((startTime + playTime) / 1000);` - brackets missing...

Reply:
nope, forget about, sorry for the noise.